pkg/steps/output_image_tag.go

```diff
@@ -143,15 +147,18 @@ func (s *outputImageTagStep) namespace() string {
 	return s.jobSpec.Namespace()
 }
 
-func (s *outputImageTagStep) imageStreamTag(fromImage string) *imagev1.ImageStreamTag {
+func (s *outputImageTagStep) imageStreamTag(fromImage string, referencePolicy imagev1.TagReferencePolicyType) *imagev1.ImageStreamTag {
+	if referencePolicy == "" {
+		referencePolicy = imagev1.LocalTagReferencePolicy
+	}
 	return &imagev1.ImageStreamTag{
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      fmt.Sprintf("%s:%s", s.config.To.Name, s.config.To.Tag),
 			Namespace: s.namespace(),
 		},
 		Tag: &imagev1.TagReference{
 			ReferencePolicy: imagev1.TagReferencePolicy{
-				Type: imagev1.LocalTagReferencePolicy,
+				Type: referencePolicy,
```

Comment:
Can you elaborate more on why the output image tag step needs to be updated too?

Reply:
we do not want the imageStream hardcoded to pull from Local and should have an option to pull from source